### DISCUSSION TITLE
skywalking metrics exporter “EventType ” means？

### DISCUSSION THREAD

**Discussion by hasaiki123** [apache/skywalking#12196] [Q&A]
when I create a grpc server，I can receive metrics send by skywalking metrics exporter，but I found the received metrics value shows different with the metrics in db，I doubt。
And I found the EventType in oap-server/exporter/src/main/proto/metric-exporter.proto：
`enum EventType {
    // The metrics aggregated in this bulk, not include the existing persistent data.
    INCREMENT = 0;
    // Final result of the metrics at this moment.
    TOTAL = 1;
}`
I cannot understand totally about EventType，If I need filter “TOTAL”？

**Comment by wu-sheng** [comment]
Total mean aggregated data for this minute, as you will receive more than once per minute.

**Comment by hasaiki123** [comment]
But I found the received metrics value shows different with the metrics in db，this is correct？
I think exported metrics should be same as metrics stored db。

**Reply by wu-sheng** [reply]
They are from the same place(objects).

I doubt they are different.

**Reply by wu-sheng** [reply]
You have to debug and address this, we process this in a consistent way. Would not change the values.

**Reply by hasaiki123** [reply]
ok，I try。
